main.js

```diff
@@ -0,0 +1,25 @@
+const email = document.querySelector(".email");
```

Comment:
```suggestion
const email = document.querySelector('.email');
```
U .js fajlovima cemo koristiti jednostruke navodnike a dvostruki ce ici samo za html attribute-e